src/environments/environment.ts

```diff
@@ -3,7 +3,12 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  googleApiUrl: "www.googleapis.com",
+  googleCalendarId: "ntuoss@gmail.com",
+  googleApiKey: "AIzaSyDWzoB1A_zTDM9eVClHKkwR5IT3UOVeats",
+  eventbriteToken: "JH5YG6RWF7BDGBLRGPBJ",
+  eventbriteUrl: "www.eventbriteapi.com"
```

Comment:
Suggest we rename `eventbriteUrl` to `eventbriteApiUrl` to be consistent with `googleApiUrl`. We should also use a different mechanism to store these configuration as it may lead to a security breach.

Reply:
Not sure if this is overkill, but we might as well integrate [travis](https://travis-ci.org/). There are a [few options](https://docs.travis-ci.com/user/environment-variables/) for us to add environment variables safely. Seems that we can use this to execute automatic builds so we don't need to do `npm build` everytime.

Reply:
May need to add to issues for enhancement